<ui version="4.0" >
 <class>grubWindow</class>
 <widget class="QMainWindow" name="grubWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Grub Kurulumu - Anatolya Kurulum</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" >#grubWindow { 
	background-image: url(:/artalan/artalan.png); }</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="1" column="0" colspan="3" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string/>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>40</y>
         <width>451</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; color:#00aaff;">Sistem önyükleyicisini (Grub) kurmak istiyormusunuz ?&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>160</y>
         <width>431</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title" >
        <string/>
       </property>
       <widget class="QRadioButton" name="noInstallGrubCheck" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>100</y>
          <width>95</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text" >
         <string>Hayır</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="installGrubCheck" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>60</y>
          <width>341</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text" >
         <string>Evet (Önerilen)</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="nextButton" >
      <property name="text" >
       <string>İleri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="backButton" >
      <property name="text" >
       <string>Geri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../images/anatolyaimages.qrc" />
 </resources>
 <connections/>
</ui>
